packages/vite-utils/README.md

```diff
@@ -0,0 +1,32 @@
+# vite-utils
```

Comment:
Whoa this was a big lift, sorry! Works great though and I think it's reasonable to have a package like this. Do you anticipate the TS compiling as being easy to maintain? Introducing a big support burden here just to avoid duplicate code would be a notable tradeoff.

Reply:
I added the compile step to the [postinstall script](https://github.com/khelif96/ui/blob/ed3b0f15f97f2793c60d4b3391c9789b5849155d/package.json#L18). So it should happen automatically after installing dependencies. I don't anticipate us changing this directory much so I think the support burden would be any more than running `yarn install`.